### DISCUSSION TITLE
Replies on BlueSky from bridged Mastodon post

### DISCUSSION THREAD

**Discussion by zaphbbrox** [snarfed/bridgy-fed#1105] [Q&A] [answered]
Maybe I'm misunderstanding Bridgy, but I thought when someone replies on Bluesky to the post I wrote on Mastodon, I would see it in Mastodon.
[bridged post on BS](https://bsky.app/profile/zaph0d.norden.social.ap.brid.gy/post/3ktzuofockor2)
[original post on Mastodon](https://norden.social/@zaph0d/112553584279715823)

**Comment by TomCasavant** [comment, marked as answer]
The replies will be bridged if your bluesky account follows the @ap.brid.gy account- I think *eventually* bluesky will be opt-out but don't quote me on that.

**Reply by snarfed** [reply]
@TomCasavant is right on both counts! (At least, on the second one, Bluesky _may_ eventually be opt out.)

**Comment by zaphbbrox** [comment]
Thanks, but how I get the bridged Bluesky account to follow @ap.brid.gy, since I don't have any credentials?
Or does every Bluesky account who replies follow @ap.brid.gy to have its reply shown in the original Mastodon post?

**Reply by snarfed** [reply]
The latter! At least for now.
